rfcs/single_test.md

```diff
@@ -0,0 +1,301 @@
+# RFC: `single_test()` opt-in for single-page tests
+
+## Summary
+
+Single-page tests are simple testharness.js tests that just do assertions and finally call `done()`. They reduce boilerplate as no `test()` or `async_test()` is needed. Callbacks also need not be wrapped with `step_func()` as any error simply fails the test. See [simple sync test](https://github.com/web-platform-tests/wpt/blob/ded00e006a083cacc108e8a1e92963fe7b15de4e/mediacapture-streams/MediaDevices-SecureContext.html) and [simple async test](https://github.com/web-platform-tests/wpt/blob/87d9cd16b9b4992649dc2ea1ecb3261f163e9184/html/webappapis/timers/negative-setinterval.html) for real examples.
+
+Unfortunately, the rules for single-page tests are subtle and many tests accidentally enter this mode. This leads to less consistent results cross-browser, see [FileAPI example](https://wpt.fyi/results/FileAPI/url/url-format.any.worker.html?run_id=312160003&run_id=306970005&run_id=321820002&run_id=319900004).
```

Comment:
```suggestion
Unfortunately, the rules for single-page tests are subtle and many tests accidentally enter this mode. This leads to less-consistent results cross-browser, see [FileAPI example](https://wpt.fyi/results/FileAPI/url/url-format.any.worker.html?run_id=312160003&run_id=306970005&run_id=321820002&run_id=319900004).
```

Reply:
You're the native speaker and I know there's some variation in how much people hyphenate, but I believe this and "more complex rules" below are grammatically okay. I'll resolve these, and apply the other fixes.

Reply:
> You're the native speaker and I know there's some variation in how much people hyphenate, but I believe this and "more complex rules" below are grammatically okay

Yeah, they are just judgement calls. The reason I usually suggest them is to eliminate ambiguity. There is actually a (somewhat subtle) difference between "more complex rules", meaning additional rules that are (absolutely) complex, and "more-complex rules", meaning rules that have relatively more complexity than normal rules.